tox.ini

```diff
@@ -0,0 +1,10 @@
+[tox]
+envlist =
+    py27,py36
```

Comment:
let's also test 3.4 and 3.5 and allow missing interpreters (similar to pyanalyze's tox.ini)

Reply:
Why do you want to allow missing interpreters?

Reply:
okay. I misunderstood the value of this option. added.